dinghy-lib/src/apple/device.rs

```diff
@@ -115,11 +124,36 @@ impl IosDevice {
         envs: &[&str],
         debugger: bool,
     ) -> Result<()> {
-        let mut command = process::Command::new("ios-deploy");
-        command.args(&["-i", &self.id, "-b", &app_path, "-m"]);
-        command.args(&["-a", &args.join(" ")]);
-        command.args(&["-s", &envs.join(" ")]);
-        command.arg(if debugger { "-d" } else { "-I" });
+        let mut command = if self.version.starts_with("17") {
+            let signing = xcode::look_for_signature_settings(&self.id)?
+                .pop()
+                .ok_or_else(|| anyhow!("no signing identity found"))?;
+            let app_id = signing
+                .name
+                .split(" ")
+                .last()
+                .ok_or_else(|| anyhow!("no app id ?"))?;
```

Comment:
From what I can tell, `devicectl` requires using the app bundle id. It fails when just `Dinghy` is used.

Ideally, this would just use the already called [`xcode::look_for_signature_settings` that was used in `make_app`](https://github.com/sonos/dinghy/blob/121354ff757f366401b279d055e40acd1b95b46d/dinghy-lib/src/apple/device.rs#L71-L78). I every way I looked, this required modifying [`BuildBundle`](https://github.com/sonos/dinghy/blob/121354ff757f366401b279d055e40acd1b95b46d/dinghy-lib/src/lib.rs#L270-L276) or the [`Device` trait](https://github.com/sonos/dinghy/blob/121354ff757f366401b279d055e40acd1b95b46d/dinghy-lib/src/lib.rs#L145-L167) both seemed worse than this option.